### PR TITLE
Theme refactor

### PR DESCRIPTION
**Pull request by craigh**
| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| New feature?      | yes
| BC breaks?        | no
| Deprecations?     | no
| Tests pass?       | yes
| Fixed tickets     | -
| Refs tickets      | #1753
| License           | MIT
| Doc PR            | -
| Changelog updated | yes

### CODE REVIEW

File: src/themes/BootstrapTheme/Resources/public/css/style.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
 body {
-    margin-top: 60px;
```
Comment (cmfcmf) [inline src/themes/BootstrapTheme/Resources/public/css/style.css]: :+1:

File: src/themes/BootstrapTheme/Resources/views/modules/ZikulaThemeModule/moduleheader.html.twig [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                        <span class="icon-bar"></span>
+                    </button>
+                    {% if isset(image) && image %}
+                        <a class="navbar-brand" href="{% if titlelink %}{{ titlelink }}{% else %}#{% endif %}" title="{{ title }}"><img class="img-responsive" alt="" src="{{ image }}" /></a>
+                    {% endif %}
+                    <a href="{% if titlelink %}{{ titlelink }}{% else %}#{% endif %}" class="navbar-brand" title="{{ title }}">{% if truncated %}{{ titletruncated }}{% else %}{{ title }}{% endif %}</a>
+                </div>
+                <div class="collapse navbar-collapse" id="userheader-div">
+                    {% if truncated %}<h4>{{ title }}</h4>{% endif %}
+                    {modulelinks modname=$modname type=$type menuclass='nav navbar-nav'}
```
Comment (cmfcmf) [inline src/themes/BootstrapTheme/Resources/views/modules/ZikulaThemeModule/moduleheader.html.twig]: ?

File: src/themes/BootstrapTheme/Resources/views/modules/ZikulaUsersModule/User/main.html.twig [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        <a href="{{ accountLink.url }}">
+            <div class="media">
+                <div class="media-left">
+                {% if modvars.ZikulaUsersModule.accountdisplaygraphics == 1 %}
+                    {% if isset(accountLink.set) && !empty(accountLink.set) %}
+                        {% set iconset = accountLink.set %}
+                    {% else %}
+                        {% set iconset = 'null' %}
+                    {% endif %}
+                    {img src=$accountLink.icon modname=$accountLink.module set=$iconset class="media-object"}
```
Comment (cmfcmf) [inline src/themes/BootstrapTheme/Resources/views/modules/ZikulaUsersModule/User/main.html.twig]: ?

Comment (craigh) [inline src/themes/BootstrapTheme/Resources/views/modules/ZikulaUsersModule/User/main.html.twig]: anything in a `/modules` dir is an override and not used until the module itself is ported to twig so, they can be left until then.

**Pull request merged**